Dockerfile

```
@@ -15,7 +15,7 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-ARG GO_VERSION=1.20.7
+ARG GO_VERSION=1.21
```

```suggestion
ARG GO_VERSION=1.21.0
```

We've generally been pinning to the patch as well, right? 

That said...I'm fine with using the rolling tag (`1.21`) if you want, despite the `Host` issues recently, Go patch upgrades are generally very safe IMO, so up to you!